web/src/routes/categories/components/CategoryList.js

```diff
@@ -51,7 +51,7 @@ class CategoryList extends React.PureComponent<PropsType> {
         {category?.title && <Caption title={category.title} />}
         {category?.content && <RemoteContent dangerouslySetInnerHTML={{ __html: category.content }}
                                    onInternalLinkClick={onInternalLinkClick} />}
-        {category?.content && category.lastUpdate &&
+        {category?.content && category.lastUpdate && formatter &&
```

Comment:
That is strange, formatter should not be optional I think.

Reply:
I think this is fine. It is not used in the category list used for the search page.